db/migrate/20230816134617_create_likes.rb

```diff
@@ -0,0 +1,10 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :user, foreign_key: true
```

Comment:
- [ ] Great job so far @Enning94 There are just a few changes I will suggest you implement and you'll be on your way to the next milestone :wink: 
- [ ] Per the project requirements, `Table and column names should match the ERD diagram` and with this implementation, the column created in the `likes table` will be `user_id` and not `author_id` as seen in the ERD diagram  :+1: So kindly fix this :+1: 
- [ ] You can do something like this instead: 
```ruby
t.references :author, foreign_key: { to_table: 'users' }
```